tests/test_utils.py

```diff
@@ -19,13 +26,7 @@ def test_torch_round_trip(batch_size):
     d6 = R_to_rot6d(R_torch)
     R_recon = rot6d_to_R(d6)
 
-    np.testing.assert_allclose(
-        R_torch.cpu().numpy(),
-        R_recon.cpu().numpy(),
-        rtol=1e-5,
-        atol=1e-6,
-        err_msg=f"Torch round-trip failed for batch size {batch_size}",
-    )
+    assert_allclose(R_torch.cpu(), R_recon.cpu(), tol=TOL)
```

Comment:
A bit of a shame to loose the debugging error messages. You could have extended `assert_allclose` to support optional argument `err_msg`, but it is up to you.